File: app/benchmark/generic.go
package main

import (
  "fmt"
)

func main() {
  result := 0

  for a := 0; a < 1000; a++ {
    for i := 0; i < 100000; i++ {
      for j := 0; j < 100; j++ {
        result = i;
      }
    }
  }

  fmt.Println("done: ", result)
}

File: app/benchmark/generic.c
int main (
    int argc,
    char** argv
)
{
  int result = 0;

  for (int a = 0; a < 1000; a++) {
    for (int i = 0; i < 100000; i++) {
      for (int j = 0; j < 100; j++) {
        result = i;
      }
    }
  }

  fprintf(stderr, "done: %i\n", result);

  return 0;
}
